query-engine/driver-adapters/js/adapter-neon/README.md

```diff
@@ -2,4 +2,70 @@
 
 Prisma driver adapter for [Neon Serverless Driver](https://github.com/neondatabase/serverless).
 
-See https://github.com/prisma/prisma/releases/tag/5.4.0 for details.
+See https://github.com/prisma/prisma/releases/tag/5.4.0 and https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV for details.
+
+The following usage tutorial is valid for Prisma 5.4.2 and later versions.
+
+## How to install
+
+After [creating your database on Neon](https://neon.tech/docs/get-started-with-neon/setting-up-a-project), you'll need to install the `@prisma/adapter-neon` driver adapter, Neon’s serverless database driver `@neondatabase/serverless`, and `ws` to set up a WebSocket connection for use by Neon.
+
+```sh
+npm install @prisma/adapter-neon
+npm install @neondatabase/serverless
+npm install ws
+```
+
+Make sure your [Neon database connection string](https://neon.tech/docs/connect/connect-from-any-app) is copied over to your `.env` file. The connection string will start with `postgres://`.
+
+```env
+# .env
+DATABASE_URL="postgres://..."
+```
+
+You can now reference this environment variable in your `schema.prisma` datasource. Make sure you also include the `driverAdapters` Preview feature.
```

Comment:
I see, maybe in the case of turso, the text was wrong due to copypasta?

Reply:
🍝